### PR TITLE
markup with components (part 2)

### PR DESCRIPTION
**Pull request by Arzamas-ka**
Implement markup and styles for 

- “Add movie”
addMovie

- “Edit movie”
editMovie

- “Delete movie”
DeleteMovie

No need to implement real API calls. Only add pages with mocked data. No need to implement hooks in this task.

### CODE REVIEW

File: src/components/AddMovie/index.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+const selectOptions = [
+  { id: 1, name: 'Action & Adventure' },
+  { id: 2, name: 'Drama, Biography, Music' },
+  { id: 3, name: 'Oscar winning movie' },
+];
+
+const AddMovie: FC = () => {
+  const [values, setValues] = useState(initialValues);
+
+  const handleOnChange = (event) => {
```
Comment (osonovaakylai) [inline src/components/AddMovie/index.tsx:33]: ES6 features, could be descructured here

File: src/components/Calendar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import DatePicker from 'react-datepicker';
+
+import { Calendar as CalendarWrapper, CalendarIcon } from './style';
+
+import Input from 'components/Input';
+
+const Calendar: FC = () => {
+  const [startDate, setStartDate] = useState(null);
+
+  const ExampleCustomInput = forwardRef(({ value, onClick }: any, ref: any) => (
```
Comment (osonovaakylai) [inline src/components/Calendar/index.tsx:11]: It would be better to move it into separate file

File: src/components/Search/index.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  const handleOnChange = (event: FormEvent<HTMLInputElement>): void => {
+    console.log('input: ', event.currentTarget.value);
+
+    setValue(event.currentTarget.value);
+  };
+
+  return (
+    <SearchStyled>
+      <h2>Find your movie</h2>
+      <form onSubmit={(event: SyntheticEvent) => event.preventDefault()}>
```
Comment (osonovaakylai) [inline src/components/Search/index.tsx:20]: Consider using Formik in the future. 
https://github.com/formium/formik
This package is widely used for creating forms

File: src/components/AddMovie/index.tsx (hunk 1 above)
Comment (Arzamas-ka) [reply src/components/AddMovie/index.tsx:33]: fixed

File: src/components/Calendar/index.tsx (hunk 2 above)
Comment (Arzamas-ka) [reply src/components/Calendar/index.tsx:11]: This custom Input is a piece of DatePicker and using only in one place.
https://reactdatepicker.com/#example-custom-input

File: src/components/Search/index.tsx (hunk 3 above)
Comment (Arzamas-ka) [reply src/components/Search/index.tsx:20]: Ok, This package will use in Task 7

image

**Review by osonovaakylai — APPROVED** [review]

**Review by osonovaakylai — APPROVED** [review]

**Pull request merged**